### DISCUSSION TITLE
czg 是不是可以直接用 .commitlintrc.ts ？

### DISCUSSION THREAD

**Discussion by yuzheng14** [Zhengqbbb/cz-git#81] [Q&A] [answered]
cz-git 的文档里说可以配合 commitlint 的配置使用，并且对 ts 的兼容性不好。但是 czg 的文档里没有提这个事，而且没法通过 `/** @type import('cz-git').UserConfig */` 来获取类型提示，是不是可以直接用 .commitlintrc.ts ？

**Comment by Zhengqbbb** [comment, marked as answer]
> cz-git 的文档里说可以配合 commitlint 的配置使用，并且对 ts 的兼容性不好。但是 czg 的文档里没有提这个事，而且没法通过 `/** @type import('cz-git').UserConfig */` 来获取类型提示，是不是可以直接用 .commitlintrc.ts ？

czg 内没有包含 `ts` 的类型提示，下一个版本可以考虑添加，如果需要提示的话的话目前可以通过安装一个 `cz-git` 依赖

> 是不是可以直接用 .commitlintrc.ts 

不行

<img width="747" alt="image" src="https://user-images.githubusercontent.com/40693636/203674902-f88d0a8f-2b7a-46b2-a72f-fed129afce7d.png">
